src/config.ts

```diff
@@ -20,12 +21,24 @@ const loadAirnodeConfig = () => {
   return config;
 };
 
+const loadAirkeeperConfig = (): AirkeeperConfig => {
+  const configPath = path.resolve(__dirname, '..', '..', 'config', `airkeeper.json`);
+  const airkeeperConfig: AirkeeperConfig = JSON.parse(fs.readFileSync(configPath, 'utf8'));
+
+  const validationOutput = validateConfig(configSchema, airkeeperConfig);
+  if (!validationOutput.success) {
+    throw new Error(`Invalid Airkeeper configuration file: ${JSON.stringify(validationOutput.error, null, 2)}`);
+  }
+
+  return validationOutput.data;
+};
+
 const parseConfig = <T>(filename: string): T => {
   const configPath = path.resolve(__dirname, '..', '..', 'config', `${filename}.json`);
   return JSON.parse(fs.readFileSync(configPath, 'utf8'));
 };
 
-const mergeConfigs = (airnodeConfig: node.Config, airkeeperConfig: Config): Config => {
+const mergeConfigs = (airnodeConfig: node.Config, airkeeperConfig: AirkeeperConfig): Config => {
```

Comment:
Can you change the type of `Config` (the resulting merged one) to reflect the new types? I mean use the ones generated by zod.